dd-sdk-android/src/main/kotlin/com/datadog/android/error/internal/CrashReportFilePersistenceStrategy.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.error.internal
+
+import android.content.Context
+import com.datadog.android.core.internal.domain.PayloadDecoration
+import com.datadog.android.core.internal.persistence.file.advanced.FeatureFileOrchestrator
+import com.datadog.android.core.internal.persistence.file.batch.BatchFilePersistenceStrategy
+import com.datadog.android.core.internal.privacy.ConsentProvider
+import com.datadog.android.core.internal.utils.sdkLogger
+import com.datadog.android.log.Logger
+import com.datadog.android.log.internal.domain.Log
+import com.datadog.android.log.internal.domain.LogSerializer
+import java.util.concurrent.ExecutorService
+
+internal class CrashReportFilePersistenceStrategy(
```

Comment:
I think you forgot about the `NdkNetworkInfoFileStrategy` or maybe it is in future commits ?

Reply:
This class replaces the `NdkNetworkInfoFileStrategy` and `NdkUserInfoFileStrategy`, since they had the exact same implementation.